_posts/2020-02-10-hhvm-4.44.markdown

```diff
@@ -0,0 +1,56 @@
+---
+title: "HHVM 4.44"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.44 is released! This release marks the end of support for 4.37;
+HHVM 4.38&ndash;4.43 remain supported, as do the 4.8 and 4.32 LTS releases.
+
+# Breaking Changes
+
+- the `__Deprecated` attribute is no longer permitted on any types, such as
+  classes, enums, or type aliases; it had no effect in these positions.
+- `(int) NAN` is now the minimum 64-bit signed integer, instead of 0.
+- square bracket literals for PHP arrays (e.g. `['foo', 'bar']`) are now a parse
+  error.
```

Comment:
maybe mention that array() is still supported but recommend using Hack arrays instead